### PR TITLE
add track extras for run 2 + store SPD tracklet flag for free vs. shared clusters

### PR DESCRIPTION
**Pull request by maciacco**
see also https://github.com/AliceO2Group/AliceO2/pull/13420

### CODE REVIEW

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 1]
```diff
@@ -2031,17 +2045,19 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
         tracks.fRho1PtSnp = 0;
         tracks.fRho1PtTgl = 0;
         tracks.fTPCinnerP = NAN;
-        tracks.fFlags = 0;
+        tracks.fFlags = mlt->FreeClustersTracklet(itr, 0) && mlt->FreeClustersTracklet(itr, 1);
```
Comment (jgrosseo) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:2048]: Shouldn't this be?
```suggestion
        tracks.fFlags = (mlt->FreeClustersTracklet(itr, 0) && mlt->FreeClustersTracklet(itr, 1)) ? FreeClsSPDTracklet : 0;
```

Comment (maciacco) [reply RUN3/AliAnalysisTaskAO2Dconverter.cxx]: thanks a lot! I merged it

**Comment by jgrosseo** [comment]
+test

**Comment by jgrosseo** [comment]
From my side this is fine. Waiting for the OK from @ddobrigk as well.

**Comment by ddobrigk** [comment]
Thanks @maciacco , this looks okay to me as well. Just for completeness, can you maybe present briefly at the Monday meeting this Monday (tagging @pzhristov for the sake of the agenda)? It does not have to be very long - just a short talk explaining the changes and their rationale should be enough to provide a reference, and then we can merge and go ahead. Thank you!

**Comment by maciacco** [comment]
hi @jgrosseo @ddobrigk thanks for the review. It's fine for me to present the next Monday!

**Comment by maciacco** [comment]
Dear @jgrosseo @ddobrigk I removed the number of TPC pid clusters from the run 2 extra table. This should make it easier to integrate in the future the changes Max is working on. See also https://github.com/AliceO2Group/AliceO2/pull/13420

**Comment by jgrosseo** [comment]
For me it is OK. Green light to be given by @ddobrigk

**Comment by jgrosseo** [comment]
+test

**Comment by maciacco** [comment]
dear @ddobrigk @jgrosseo @pzhristov thanks a lot for the review! Would it be possible to merge this PR in the master of AliPhysics? (Also the corresponding one for O2)

**Comment by ddobrigk** [comment]
We should merge this so it can be included in the conversions that @maciacco has started exercising... I think we only approved testing? Can we approve it in full? 

BTW @mpuccio is also expected to report on the TPC clusters for PID topic next Monday, and IMHO we should move ahead swiftly also with that change as well. Thank you!

**Comment by mpuccio** [comment]
+1

**Pull request merged**